test/build.test.js

```diff
@@ -19,6 +19,8 @@ if (process.platform === 'win32') {
 process.env.PATH = cmd_path + sep + process.env.PATH;
 process.env.NODE_PATH = path.join(__dirname,'../lib/');
 
+var jsEngine = process.jsEngine ? '-' + process.jsEngine : '';
```

Comment:
>process.jsEngine ? '-' + process.jsEngine : ''; [](start = 15, length = 47)

I would just use `process.jsEngine || 'v8'` unless you have reason of doing it this way.